components/common/Loading.tsx

```diff
@@ -0,0 +1,25 @@
+import { BounceLoader } from "react-spinners";
+import Logo from "./Logo";
+
+const Loading = () => {
+    return (
+        <div className='grid grid-cols-1 gap-4 lg:col-span-2 mt-10'>
+            <span className="flex items-center justify-center">
```

Comment:
super small issue only change if have time HAHA but maybe can center-align the text only and align the logo to the left of the text using `absolute` positioning